src/EnergyPlus/MixedAir.cc

```diff
@@ -1097,7 +1121,7 @@ void GetOutsideAirSysInputs(EnergyPlusData &state)
                 } else if (SELECT_CASE_var == "COILSYSTEM:COOLING:DX") {
                     state.dataAirLoop->OutsideAirSys(OASysNum).ComponentType_Num(CompNum) = DXSystem;
                     // set the data for 100% DOAS DX cooling coil
-                    CheckDXCoolingCoilInOASysExists(state, state.dataAirLoop->OutsideAirSys(OASysNum).ComponentName(CompNum));
+                    // CheckDXCoolingCoilInOASysExists(state, state.dataAirLoop->OutsideAirSys(OASysNum).ComponentName(CompNum));
```

Comment:
Wasn't sure if this needs to be done here (timing) but appears to be done later.